exchanges/btse/btse_wrapper.go

```diff
@@ -4,7 +4,6 @@ import (
 	"context"
```

Comment:
Since this PR is about handling of funky currencies, I've got to add an extra request for you.
I've noticed that updating tickers via REST is not successful because we are not formatting currencies:
```
[DEBUG] | REQUESTER | 14/06/2024 11:04:20 | BTSE attempt 1 request path: https://api.btse.com/spot/api/v3.2/market_summary?symbol=K_HTXUSD
[DEBUG] | REQUESTER | 14/06/2024 11:04:20 | BTSE request type: GET
[ERROR] | SYNC | 14/06/2024 11:04:21 | Failed to get REST ticker. Error: BTSE unsuccessful HTTP status code: 400 raw response: {"code":-1,"msg":"Unknown symbol","time":1718327060993,"data":null,"success":false}
```
Please do the following for `UpdateTicker`
```go
// UpdateTicker updates and returns the ticker for a currency pair
func (b *BTSE) UpdateTicker(ctx context.Context, p currency.Pair, a asset.Item) (*ticker.Price, error) {
	if p.IsEmpty() {
		return nil, currency.ErrCurrencyPairEmpty
	}
	if !b.SupportsAsset(a) {
		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, a)
	}
	fPair, err := b.FormatSymbol(p, a)
	if err != nil {
		return nil, err
	}
	ticks, err := b.GetMarketSummary(ctx, fPair, a == asset.Spot)
```
Then its all good

```
BTSE attempt 1 request path: https://api.btse.com/spot/api/v3.2/market_summary?symbol=K_HTX-USD
{{success}}
```

Reply:
Fixed 3f033d4

Not massively happy we can't (even) format a symbol without engendering some `err` risk. Tomorrow.